tests/api2/ups.py

```diff
@@ -107,19 +182,42 @@ def test_10_get_ups_driver_choice():
 
 
 @pytest.mark.parametrize('dkey', ups_dc_list)
-def test_11_check_ups_driver_choice_info_(dkey):
+def test_19_check_ups_driver_choice_info_(dkey):
     driver_choices = dkey.partition('$')[2]
```

Comment:
While we are making a few changes, let's please name this to `driver_choice`, it can confuse a reader.